### Importación de datos



In [1]:
import pandas as pd

url = "https://raw.githubusercontent.com/alura-es-cursos/challenge1-data-science-latam/refs/heads/main/base-de-datos-challenge1-latam/tienda_1%20.csv"
url2 = "https://raw.githubusercontent.com/alura-es-cursos/challenge1-data-science-latam/refs/heads/main/base-de-datos-challenge1-latam/tienda_2.csv"
url3 = "https://raw.githubusercontent.com/alura-es-cursos/challenge1-data-science-latam/refs/heads/main/base-de-datos-challenge1-latam/tienda_3.csv"
url4 = "https://raw.githubusercontent.com/alura-es-cursos/challenge1-data-science-latam/refs/heads/main/base-de-datos-challenge1-latam/tienda_4.csv"

tienda = pd.read_csv(url)
tienda2 = pd.read_csv(url2)
tienda3 = pd.read_csv(url3)
tienda4 = pd.read_csv(url4)

tienda.head()

Producto Categoría del Producto    Precio  Costo de envío  \
0  Asistente virtual           Electrónicos  164300.0          6900.0   
1    Mesa de comedor                Muebles  192300.0          8400.0   
2      Juego de mesa               Juguetes  209600.0         15900.0   
3         Microondas      Electrodomésticos  757500.0         41000.0   
4   Silla de oficina                Muebles  335200.0         20200.0   

  Fecha de Compra         Vendedor Lugar de Compra  Calificación  \
0      16/01/2021      Pedro Gomez          Bogotá             4   
1      18/05/2022  Beatriz Morales        Medellín             1   
2      15/03/2021   Juan Fernandez       Cartagena             1   
3      03/05/2022   Juan Fernandez            Cali             4   
4      07/11/2020    Maria Alfonso        Medellín             5   

       Método de pago  Cantidad de cuotas       lat       lon  
0  Tarjeta de crédito                   8   4.60971 -74.08175  
1  Tarjeta de crédito                   4   6.25184 -75.56359  
2  Tarjeta de crédito                   1  10.39972 -75.51444  
3               Nequi                   1   3.43722 -76.52250  
4               Nequi                   1   6.25184 -75.56359

In [2]:
# facturación total de la tienda

total_facturado = tienda['Precio'].sum()
print("El total de facturación de la tienda 1 es:", total_facturado )


El total de facturación de la tienda 1 es: 1150880400.0


In [3]:
tienda2

Producto   Categoría del Producto     Precio  Costo de envío  \
0     Silla de oficina                  Muebles   283600.0         15300.0   
1            Pandereta   Instrumentos musicales    76800.0          2300.0   
2            Cama king                  Muebles   443300.0         23800.0   
3            Cama king                  Muebles   906200.0         46400.0   
4     Silla de oficina                  Muebles   397800.0         19400.0   
...                ...                      ...        ...             ...   
2354      Set de ollas  Artículos para el hogar   140300.0          8100.0   
2355  Lavadora de ropa        Electrodomésticos  1220100.0         65200.0   
2356        Microondas        Electrodomésticos   797900.0         42900.0   
2357         Iphone 15             Electrónicos  2224300.0        116600.0   
2358         Bicicleta     Deportes y diversión   551000.0         27500.0   

     Fecha de Compra         Vendedor Lugar de Compra  Calificación  \
0         04/08/2021  Izabela de León        Medellín             1   
1         26/11/2020   Blanca Ramirez        Medellín             3   
2         22/10/2021  Izabela de León         Pereira             5   
3         04/05/2022    Maria Alfonso            Cali             5   
4         26/02/2022    Lucas Olivera          Bogotá             4   
...              ...              ...             ...           ...   
2354      04/12/2022  Ángel Rodriguez    Barranquilla             5   
2355      09/02/2020   Santiago Silva          Bogotá             4   
2356      27/09/2022   Blanca Ramirez          Bogotá             5   
2357      21/04/2022   Juan Fernandez        Medellín             5   
2358      03/09/2022    Maria Alfonso        Medellín             4   

          Método de pago  Cantidad de cuotas       lat       lon  
0                  Nequi                   1   6.25184 -75.56359  
1     Tarjeta de crédito                   4   6.25184 -75.56359  
2     Tarjeta de crédito                  10   4.81333 -75.69611  
3     Tarjeta de crédito                  10   3.43722 -76.52250  
4     Tarjeta de crédito                   3   4.60971 -74.08175  
...                  ...                 ...       ...       ...  
2354  Tarjeta de crédito                   2  10.96854 -74.78132  
2355  Tarjeta de crédito                   5   4.60971 -74.08175  
2356               Nequi                   1   4.60971 -74.08175  
2357  Tarjeta de crédito                   2   6.25184 -75.56359  
2358  Tarjeta de crédito                   1   6.25184 -75.56359  

[2359 rows x 12 columns]

In [4]:
tienda3

Producto  Categoría del Producto     Precio  \
0            Secadora de ropa       Electrodomésticos  1285100.0   
1               Mesa de noche                 Muebles   290100.0   
2           Guitarra acústica  Instrumentos musicales   303500.0   
3           Balón de voleibol    Deportes y diversión    58800.0   
4                     Armario                 Muebles   521100.0   
...                       ...                     ...        ...   
2354  Dashboards con Power BI                  Libros    26700.0   
2355  Dashboards con Power BI                  Libros    31800.0   
2356         Silla de oficina                 Muebles   477100.0   
2357           Mesa de centro                 Muebles   123100.0   
2358      Modelado predictivo                  Libros    63400.0   

      Costo de envío Fecha de Compra         Vendedor Lugar de Compra  \
0            66600.0      17/12/2021    Camila Rivera     Santa Marta   
1            13600.0      10/02/2021    Maria Alfonso            Cali   
2            17400.0      22/01/2023    Lucas Olivera          Bogotá   
3             3300.0      22/03/2022      Pedro Gomez    Barranquilla   
4            25900.0      12/01/2022   Blanca Ramirez          Bogotá   
...              ...             ...              ...             ...   
2354          2100.0      21/10/2022  Ángel Rodriguez          Bogotá   
2355          2200.0      23/06/2022   Blanca Ramirez         Pereira   
2356         25600.0      19/02/2023    Camila Rivera       Cartagena   
2357          4700.0      19/09/2021   Blanca Ramirez          Bogotá   
2358          9400.0      24/05/2022  Mariana Herrera        Medellín   

      Calificación      Método de pago  Cantidad de cuotas       lat       lon  
0                4  Tarjeta de crédito                   4  11.24079 -74.19904  
1                3  Tarjeta de crédito                   1   3.43722 -76.52250  
2                5  Tarjeta de crédito                   3   4.60971 -74.08175  
3                4  Tarjeta de crédito                   6  10.96854 -74.78132  
4                1  Tarjeta de crédito                   1   4.60971 -74.08175  
...            ...                 ...                 ...       ...       ...  
2354             5  Tarjeta de crédito                   1   4.60971 -74.08175  
2355             5  Tarjeta de crédito                   1   4.81333 -75.69611  
2356             5  Tarjeta de crédito                   3  10.39972 -75.51444  
2357             4  Tarjeta de crédito                   8   4.60971 -74.08175  
2358             5  Tarjeta de crédito                  10   6.25184 -75.56359  

[2359 rows x 12 columns]

In [5]:
tienda4

Producto   Categoría del Producto     Precio  Costo de envío  \
0            Lavavajillas        Electrodomésticos  1084800.0         56000.0   
1                Cama box                  Muebles   473800.0         25400.0   
2     Balón de baloncesto     Deportes y diversión    23700.0             0.0   
3     Modelado predictivo                   Libros    57700.0          1300.0   
4             Auriculares             Electrónicos   138900.0          5600.0   
...                   ...                      ...        ...             ...   
2353      Mesa de comedor                  Muebles   227700.0         12600.0   
2354     Silla de oficina                  Muebles   342000.0         18600.0   
2355         Vaso térmico     Deportes y diversión    36700.0          2400.0   
2356         Set de ollas  Artículos para el hogar   151800.0          8300.0   
2357   Cuerda para saltar     Deportes y diversión    11200.0          1300.0   

     Fecha de Compra         Vendedor Lugar de Compra  Calificación  \
0         11/11/2022    Rafael Acosta          Bogotá             4   
1         03/01/2020  Izabela de León          Cúcuta             5   
2         25/12/2022    Maria Alfonso        Medellín             5   
3         23/11/2021    Lucas Olivera        Medellín             3   
4         25/04/2022   Blanca Ramirez          Bogotá             4   
...              ...              ...             ...           ...   
2353      12/02/2020    Camila Rivera        Medellín             5   
2354      17/05/2022    Felipe Santos     Bucaramanga             5   
2355      03/05/2022    Camila Rivera          Bogotá             5   
2356      15/06/2020   Blanca Ramirez            Cali             5   
2357      17/11/2021    Felipe Santos          Bogotá             5   

          Método de pago  Cantidad de cuotas      lat       lon  
0     Tarjeta de crédito                   3  4.60971 -74.08175  
1     Tarjeta de crédito                   2  7.89391 -72.50782  
2                  Nequi                   1  6.25184 -75.56359  
3                  Nequi                   1  6.25184 -75.56359  
4     Tarjeta de crédito                   7  4.60971 -74.08175  
...                  ...                 ...      ...       ...  
2353  Tarjeta de crédito                   1  6.25184 -75.56359  
2354  Tarjeta de crédito                   3  7.12539 -73.11980  
2355  Tarjeta de crédito                   1  4.60971 -74.08175  
2356  Tarjeta de crédito                   1  3.43722 -76.52250  
2357  Tarjeta de crédito                   1  4.60971 -74.08175  

[2358 rows x 12 columns]

# **1. Análisis de facturación**



# **Total de las Ventas por Tienda y por años**

In [6]:
# TOTAL DE VENTAS DE CADA TIENDA POR AÑO

# Adiciona una columna 'Tienda' a cada DataFrame para identificar la tienda
tienda['Tienda'] = 'Tienda 1'
tienda2['Tienda'] = 'Tienda 2'
tienda3['Tienda'] = 'Tienda 3'
tienda4['Tienda'] = 'Tienda 4'

# Concatenar todos los Dataframes
all_tiendas = pd.concat([tienda, tienda2, tienda3, tienda4])

# Convierte 'Fecha' en fecha y hora y extrae el año
all_tiendas['Fecha de Compra'] = pd.to_datetime(all_tiendas['Fecha de Compra'])
all_tiendas['Año'] = all_tiendas['Fecha de Compra'].dt.year

# Agrupa por 'Tienda' y 'Año' y suma el 'Precio'
ventas_por_tienda_año = all_tiendas.groupby(['Tienda', 'Año'])['Precio'].sum().reset_index()

print("Total de ventas de cada tienda por año:")
print(ventas_por_tienda_año)

Total de ventas de cada tienda por año:
      Tienda   Año       Precio
0   Tienda 1  2020  368933200.0
1   Tienda 1  2021  362120800.0
2   Tienda 1  2022  316565600.0
3   Tienda 1  2023  103260800.0
4   Tienda 2  2020  320466600.0
5   Tienda 2  2021  351215500.0
6   Tienda 2  2022  358226500.0
7   Tienda 2  2023   86434900.0
8   Tienda 3  2020  321707500.0
9   Tienda 3  2021  362954400.0
10  Tienda 3  2022  350438300.0
11  Tienda 3  2023   62919400.0
12  Tienda 4  2020  330847700.0
13  Tienda 4  2021  347823600.0
14  Tienda 4  2022  302221500.0
15  Tienda 4  2023   57482900.0


C:\Users\Edison Sanchez\AppData\Local\Temp\ipykernel_10316\4159372776.py:13: UserWarning: Parsing dates in %d/%m/%Y format when dayfirst=False (the default) was specified. Pass `dayfirst=True` or specify a format to silence this warning.
  all_tiendas['Fecha de Compra'] = pd.to_datetime(all_tiendas['Fecha de Compra'])


# **La tienda 1 registra por año un mayor valor de ventas comparado con las demás tiendas**

# **Total Facturado por Tienda entre los años 2020 a 2023**

In [8]:
# FACTURACION TOTAL TIENDA 1
total_facturado_tienda1 = tienda['Precio'].sum()
print("El total de facturación de la tienda 1 es:", total_facturado_tienda1 )

# FACTURACION TOTAL TIENDA 2
total_facturado_tienda2 = tienda2['Precio'].sum()
print("El total de facturación de la tienda 2 es:", total_facturado_tienda2 )

# FACTURACION TOTAL TIENDA 3
total_facturado_tienda3 = tienda3['Precio'].sum()
print("El total de facturación de la tienda 3 es:", total_facturado_tienda3 )

# FACTURACION TOTAL TIENDA 4
total_facturado_tienda4 = tienda4['Precio'].sum()
print("El total de facturación de la tienda 4 es:", total_facturado_tienda4 )

El total de facturación de la tienda 1 es: 1150880400.0
El total de facturación de la tienda 2 es: 1116343500.0
El total de facturación de la tienda 3 es: 1098019600.0
El total de facturación de la tienda 4 es: 1038375700.0


# **La Tienda que Mas Facturo o Vendio es la Tienda 1**

In [9]:
# LA TIENDA QUE MAS VENDIO

facturacion_tiendas = {
    "Tienda 1": total_facturado_tienda1,
    "Tienda 2": total_facturado_tienda2,
    "Tienda 3": total_facturado_tienda3,
    "Tienda 4": total_facturado_tienda4,
}

tienda_mas_facturo = max(facturacion_tiendas, key=facturacion_tiendas.get)
monto_mas_facturado = facturacion_tiendas[tienda_mas_facturo]

print(f"La tienda que más facturó es la {tienda_mas_facturo} con un total de $ {monto_mas_facturado:,.2f}".replace(',', '.'))


La tienda que más facturó es la Tienda 1 con un total de $ 1.150.880.400.00


# **La Tienda que Menos Facturo o Vendio es la Tienda 4**

In [10]:
# LA TIENDA QUE MENOS VENDIO

facturacion_tiendas = {
    "Tienda 1": total_facturado_tienda1,
    "Tienda 2": total_facturado_tienda2,
    "Tienda 3": total_facturado_tienda3,
    "Tienda 4": total_facturado_tienda4,
}

tienda_menos_facturo = min(facturacion_tiendas, key=facturacion_tiendas.get)
monto_menos_facturado = facturacion_tiendas[tienda_menos_facturo]

print(f"La tienda que menos facturó es la {tienda_menos_facturo} con un total de $ {monto_menos_facturado:,.2f}".replace(',', '.'))

La tienda que menos facturó es la Tienda 4 con un total de $ 1.038.375.700.00


# **2. Ventas por categoría**

# **Producto más popular por tienda (basado en la categoría con mayores ventas)**

In [12]:
# VENTAS POR CATEGORIA DE PRODUCTOS MAS POPULARES POR TIENDA

# Tienda 1
ventas_por_categoria_tienda1 = tienda.groupby('Categoría del Producto')['Precio'].sum().sort_values(ascending=False)
print("Ventas por categoría en Tienda 1:\n", ventas_por_categoria_tienda1)

# Tienda 2
ventas_por_categoria_tienda2 = tienda2.groupby('Categoría del Producto')['Precio'].sum().sort_values(ascending=False)
print("\nVentas por categoría en Tienda 2:\n", ventas_por_categoria_tienda2)

# Tienda 3
ventas_por_categoria_tienda3 = tienda3.groupby('Categoría del Producto')['Precio'].sum().sort_values(ascending=False)
print("\nVentas por categoría en Tienda 3:\n", ventas_por_categoria_tienda3)

# Tienda 4
ventas_por_categoria_tienda4 = tienda4.groupby('Categoría del Producto')['Precio'].sum().sort_values(ascending=False)
print("\nVentas por categoría en Tienda 4:\n", ventas_por_categoria_tienda4)

# Producto más popular por tienda (basado en la categoría con mayores ventas)
print("\nCategoría de producto más popular por tienda:")
print("Tienda 1:", ventas_por_categoria_tienda1.index[0])
print("Tienda 2:", ventas_por_categoria_tienda2.index[0])
print("Tienda 3:", ventas_por_categoria_tienda3.index[0])
print("Tienda 4:", ventas_por_categoria_tienda4.index[0])

Ventas por categoría en Tienda 1:
 Categoría del Producto
Electrónicos               429493500.0
Electrodomésticos          363685200.0
Muebles                    187633700.0
Instrumentos musicales      91299000.0
Deportes y diversión        39290000.0
Juguetes                    17995700.0
Artículos para el hogar     12698400.0
Libros                       8784900.0
Name: Precio, dtype: float64

Ventas por categoría en Tienda 2:
 Categoría del Producto
Electrónicos               410831100.0
Electrodomésticos          348567800.0
Muebles                    176426300.0
Instrumentos musicales     104990300.0
Deportes y diversión        34744500.0
Juguetes                    15945400.0
Artículos para el hogar     14746900.0
Libros                      10091200.0
Name: Precio, dtype: float64

Ventas por categoría en Tienda 3:
 Categoría del Producto
Electrónicos               410775800.0
Electrodomésticos          329237900.0
Muebles                    201072100.0
Instrumentos musicales   

# **La categoria más Popular o vendida por Tienda son los productos Electrónicos**

# **3. Calificación promedio de la tienda**


In [14]:
# Promedio de calificación de los clientes por tienda

# Calificación promedio de la tienda
calificacion_promedio_tienda1 = tienda['Calificación'].mean()
calificacion_promedio_tienda2 = tienda2['Calificación'].mean()
calificacion_promedio_tienda3 = tienda3['Calificación'].mean()
calificacion_promedio_tienda4 = tienda4['Calificación'].mean()

print(f"Calificación promedio de la Tienda 1: {calificacion_promedio_tienda1:.2f}")
print(f"Calificación promedio de la Tienda 2: {calificacion_promedio_tienda2:.2f}")
print(f"Calificación promedio de la Tienda 3: {calificacion_promedio_tienda3:.2f}")
print(f"Calificación promedio de la Tienda 4: {calificacion_promedio_tienda4:.2f}")

Calificación promedio de la Tienda 1: 3.98
Calificación promedio de la Tienda 2: 4.04
Calificación promedio de la Tienda 3: 4.05
Calificación promedio de la Tienda 4: 4.00


# **La Tienda con mayor promedio de calificación por los Clientes; es la Tienda 3 con (4.05) y la de Menor calificación es la Tienda 1 con (3.98)**

# **4. Productos más y menos vendidos**

# **Los cinco Productos más vendidos por Tienda**

In [16]:
# LOS CINCO PRODUCTOS QUE MAS SE VENDEN POR TIENDA

# Función para encontrar los productos más vendidos por tienda
def productos_mas_vendidos(df, nombre_tienda):
    # Contar la frecuencia de cada producto
    conteo_productos = df['Producto'].value_counts()

    # Mostrar los productos más vendidos
    print(f"\nProductos más vendidos por unidad en la {nombre_tienda}:")
    print(conteo_productos.head(5)) # Mostrar los primeros 5 productos más vendidos

productos_mas_vendidos(tienda, "Tienda 1")
productos_mas_vendidos(tienda2, "Tienda 2")
productos_mas_vendidos(tienda3, "Tienda 3")
productos_mas_vendidos(tienda4, "Tienda 4")


Productos más vendidos por unidad en la Tienda 1:
Producto
Microondas          60
TV LED UHD 4K       60
Armario             60
Secadora de ropa    58
Mesa de noche       56
Name: count, dtype: int64

Productos más vendidos por unidad en la Tienda 2:
Producto
Iniciando en programación    65
Microondas                   62
Batería                      61
Guitarra acústica            58
Pandereta                    58
Name: count, dtype: int64

Productos más vendidos por unidad en la Tienda 3:
Producto
Kit de bancas      57
Mesa de comedor    56
Cama king          56
Set de ollas       55
Mesa de noche      55
Name: count, dtype: int64

Productos más vendidos por unidad en la Tienda 4:
Producto
Cama box                     62
Cubertería                   59
Dashboards con Power BI      56
Cama king                    56
Carrito de control remoto    55
Name: count, dtype: int64


# **Los cinco Productos Menos vendidos por Tienda**

In [17]:
# LOS CINCO PRODUCTOS QUE MENOS SE VENDEN POR TIENDA

# Función para encontrar los productos menos vendidos por tienda
def productos_menos_vendidos(df, nombre_tienda):
    # Contar la frecuencia de cada producto
    conteo_productos = df['Producto'].value_counts()

    # Mostrar los productos menos vendidos
    print(f"\nProductos menos vendidos por unudad en la {nombre_tienda}:")
    print(conteo_productos.tail(5)) # Mostrar los últimos 5 productos menos vendidos

productos_menos_vendidos(tienda, "Tienda 1")
productos_menos_vendidos(tienda2, "Tienda 2")
productos_menos_vendidos(tienda3, "Tienda 3")
productos_menos_vendidos(tienda4, "Tienda 4")


Productos menos vendidos por unudad en la Tienda 1:
Producto
Ciencia de datos con Python    39
Pandereta                      36
Olla de presión                35
Auriculares con micrófono      33
Celular ABXY                   33
Name: count, dtype: int64

Productos menos vendidos por unudad en la Tienda 2:
Producto
Auriculares        37
Sillón             35
Mesa de comedor    34
Impresora          34
Juego de mesa      32
Name: count, dtype: int64

Productos menos vendidos por unudad en la Tienda 3:
Producto
Guitarra eléctrica         38
Set de vasos               36
Mochila                    36
Microondas                 36
Bloques de construcción    35
Name: count, dtype: int64

Productos menos vendidos por unudad en la Tienda 4:
Producto
Refrigerador                   38
Ciencia de datos con Python    38
Guitarra acústica              37
Armario                        34
Guitarra eléctrica             33
Name: count, dtype: int64


# **5. Envío promedio por tienda**

In [18]:
# EL COSTO PROMEDIO DE ENVIO POR TIENDA

costo_envio_promedio_tienda1 = tienda['Costo de envío'].mean()
costo_envio_promedio_tienda2 = tienda2['Costo de envío'].mean()
costo_envio_promedio_tienda3 = tienda3['Costo de envío'].mean()
costo_envio_promedio_tienda4 = tienda4['Costo de envío'].mean()

print(f"Costo promedio de envío de la Tienda 1: ${costo_envio_promedio_tienda1:,.0f}".replace(',', '.'))
print(f"Costo promedio de envío de la Tienda 2: ${costo_envio_promedio_tienda2:,.0f}".replace(',', '.'))
print(f"Costo promedio de envío de la Tienda 3: ${costo_envio_promedio_tienda3:,.0f}".replace(',', '.'))
print(f"Costo promedio de envío de la Tienda 4: ${costo_envio_promedio_tienda4:,.0f}".replace(',', '.'))

Costo promedio de envío de la Tienda 1: $26.019
Costo promedio de envío de la Tienda 2: $25.216
Costo promedio de envío de la Tienda 3: $24.806
Costo promedio de envío de la Tienda 4: $23.459


# **La tienda con mayor costo promedio de envío es la tienda 1 con $26.019 y La Tienda con menor costo promedio de envío es la tienda 4 con 23.459**

Analisis de Datos.

- De los datos extraidos de los dataframes se observa que la tienda que mayor ventas realizo durante los año 2020 al 2023 es la tienda 1; con un valor total del $1.150.880.400,00 pesos y la de menor valor en ventas, fue la tienda 4 con un total de 1.038.375.700,00 pesos.

- Teniendo encuenta que la tienda 4 tiene menor valor facturado por ventas en el periodo de analisis. Las cegorias de Juguetes y Articulos para Hogar de esta tienda presentan un total de ventas mayor que las demas tiendas.

- En todas las tiendas, la categoria mas vendida fue la de productos eléctornicos.

- La Tienda con mayor promedio de calificación por los Clientes; es la Tienda 3 con (4.05) y la de Menor calificación es la Tienda 1 con (3.98).


- Los productos mas vendidos son:
  Microondas con 122 unidades en las tiendas 1 y 3
  Cama King  con 112 unidades en las tiendas 3 y 4
  Mesa de Noche con 111 unidades en las tiendas 1 y 3

Siendo asi la tienda con mayor participación en la venta de los productos mas vendidos es la tienda 3, seguida de la tienda 1.

- Los productos menos vendidos son:
  Libro de ciencia datos con Python con 77 unidades en las tiendas 1 y 4
  Guitarra eléctrica con 71 unidades en las tiendas 3 y 4
  Juego de Mesa con 32 unidades en la tienda 2

La tienda con mayor representación de productos menos vendidos es la tienda 4.

- La tienda con mayor costo promedio de envío es la tienda 1 con $26.019 pesos y La Tienda con menor costo promedio de envío es la tienda 4 con 23.459 pesos





# **Teniendo en cuenta lo anterior se concluye que la tienda con menor participación en ventas, productos menos vendidos y menor promedio de envio es la tienda 4. se aconseja vender la tienda 4. Para invertir en un nuevo proyecto o fortalecer las otras tiendas en sus estrategias de ventas, atención al cliente y la inovación de nuevos productos.**